benchopt/runner.py

```diff
@@ -216,18 +275,19 @@ def progress(id_stop_val, delta):
         return []
 
     id_stop_val = 0
-    stop_val = 1
+    stop_val = 0
```

Comment:
just call `run_one_stop_val_cached`  it ouside the loop for the initializer. This will avoid to mess things up for the `get_next`.